pkg/watcher/reconciler/dynamic/dynamic.go

```diff
@@ -572,3 +598,64 @@ func (r *Reconciler) streamLogs(ctx context.Context, o results.Object, logType,
 
 	return nil
 }
+
+// storeEvents streams logs to the API server
+func (r *Reconciler) storeEvents(ctx context.Context, o results.Object) error {
+	logger := logging.FromContext(ctx)
+	condition := o.GetStatusCondition().GetCondition(apis.ConditionSucceeded)
+	GVK := o.GetObjectKind().GroupVersionKind()
+	if !GVK.Empty() &&
+		(GVK.Kind == "TaskRun" || GVK.Kind == "PipelineRun") &&
+		condition != nil &&
+		condition.Type == "Succeeded" &&
```

Comment:
If I understand correctly, we are only storing events in case of Success. Should this include more conditions?

Reply:
It's not success but `succeeded`. It means that taskrun ran.

Reply:
I have added it to include all conditions except except when it's unkown.